app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+<ul>
+<% @users.each do |user| %>
+  <div class = "user"><li><%= user.name %>
```

Comment:
Perhaps we shouldn't include the current user in this list, so that the admin is unable to delete itself. We don't want the admin to accidentally delete it's own account and then be unable to access the dashboard.

```
@users.each do |user|
    if user != current_user
+  <%= user.username %>
+  <%= user.email %>
+  <%= link_to("Delete user", user_path(user), method: :delete, data: {confirm: "Are you sure?"}, class: "button alert round") %></li></div>
+<% end %>
+</ul>
+</div>
```